modules/cli-installing-cli-brew.adoc

```diff
@@ -15,9 +15,18 @@ For macOS, you can install the OpenShift CLI (`oc`) by using the link:https://br
 
 .Procedure
 
-* Run the following command to install the link:https://formulae.brew.sh/formula/openshift-cli[openshift-cli] package:
+* Install the link:https://formulae.brew.sh/formula/openshift-cli[openshift-cli] package by running the following command:
 +
 [source,terminal]
 ----
 $ brew install openshift-cli
 ----
+
+.Verification
+
+* After you install the the OpenShift CLI, it is available using the `oc` command:
```

Comment:
Should "OpenShift CLI" be an attribute? It was replaced with one in earlier files.

Reply:
Changing the wording of this line. It's kinda awkward anyway.